packages/vscode-extension/src/extension.mts

```diff
@@ -119,7 +119,7 @@ export async function activate(context: vsc.ExtensionContext) {
 
 		if (customCapabilities?.showCacheRoot) {
 			context.subscriptions.push(
-				vsc.commands.registerCommand('spyglassmc.showCacheRoot', async (): Promise<void> => {
+				vsc.commands.registerCommand('spyglassmc.showCacheRoot', async () => {
```

Comment:
guessing the `Promise<void>` is implied here?

Reply:
Yes. The other commands also don't have it